submissions/sean-kyle/scripts/articleView.js

```diff
@@ -65,7 +88,14 @@ articleView.handleMainNav = function() {
   //         You may need to dynamically build a selector string (concatenation???)
   //          with the correct ID, based on the data available to you on the .tab
   //          element that was clicked.
-  $('.main-nav').on(/* CODE GOES HERE */);
+  $('.main-nav').on('click', '.icon-home' , function() {
```

Comment:
Again, this works, but there's a better way to handle the situation. This works for just a couple menu elements, but imagine a store with tons of categories. You can't hardcode all of those. 

Consider the data-content attribute on the main-nav links, and how you could get at it in jQuery. (hint: .data() will help here!) Once you have that word, how would you relate it to the the section you want to show? Maybe by an id?